setup.py

```diff
@@ -208,8 +217,9 @@ def update_matplotlibrc(path):
 class BuildPy(setuptools.command.build_py.build_py):
     def run(self):
         super().run()
-        update_matplotlibrc(
-            Path(self.build_lib, "matplotlib/mpl-data/matplotlibrc"))
+        if not self.editable_mode:
```

Comment:
It surprised me to discover that when you build the `main` branch in editable mode this `run` function is not called. This branch does call the `run` function so here I am explicitly avoiding the `update_matplotlibrc` call when in editable mode to give the same behaviour as `main` branch.

Reply:
Editable (via pip/setuptools) is quite different from a regular build. I'm not an expert there, but have the feeling that if in doubt expectations on editable installs do not hold. Possibly [pep660](https://peps.python.org/pep-0660/) will change that.